### PR TITLE
added a blog by jiucl

### PR DESCRIPTION
**Pull request by ghost**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [√ ] Add Blog Header, including title, author, date.
e.g.

``` md
---
title: "'Score Your Face Photo' a ML&Wechaty practice"
author: huyingxi
---
> Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [ √] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ √] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [√ ] Add teaser image for the blog
- [ √] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-08-11-rcbot.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: "传统工业与信息化的结合"
+author: jiucl
+date: "2020-08-11 09:38:00 +0800"
+categories: project
+tags:
+  - wechaty
+---
```
Comment (lijiarui) [inline _posts/2020-08-11-rcbot.md:10]: Please add a teaser for this blog to make it more beautiful.

File: _posts/2020-08-11-rcbot.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+* 手动
+  * 添加用户为好友并拉入群聊
+* 机器人
+  * 接收消息
+  * 从消息中提取关键字，匹配产品编码和VIN码
+  * 查询产品编码和产品信息
+  * 发送产品编码和产品信息（还可以@用户？）
+
+## 代码：目前的成果
```
Comment (lijiarui) [inline _posts/2020-08-11-rcbot.md:45]: Where is this content?

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: _posts/2020-08-11-rcbot.md (hunk 2 above)
Comment (ghost) [reply _posts/2020-08-11-rcbot.md:45]: 部分位于本地文件，部分需要通过网络接口获取

Comment (lijiarui) [reply _posts/2020-08-11-rcbot.md:45]: I suggest you add more information to the blog, then this blog can provide more information for the community.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request, thanks!

**Comment by lijiarui** [comment]
Please make Github Action green first.

**Comment by lijiarui** [comment]
I suggest you add more information to the blog, then this blog can provide more information for the community.

File: _posts/2020-08-11-rcbot.md (hunk 2 above)
Comment (lijiarui) [reply _posts/2020-08-11-rcbot.md:45]: @jiucl You didn't add any content here. Please add more content here, and then I can keep review this blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request, before you giving more content, I couldn't review your blog because the blog should give the wechaty community more insight.

File: _posts/2020-08-11-rcbot.md (hunk 2 above)
Comment (ghost) [reply _posts/2020-08-11-rcbot.md:45]: I have made a new commit.Please See: https://github.com/wechaty/wechaty.js.org/pull/456/commits/963ac5af11511b00fa7ae52ddb5401f3d903c79e   .I don’t know if there is a problem with my process.

**Review by lijiarui — APPROVED** [review]
It seems good to me. @huan Please review this blog.

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty
- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room

image

**Pull request merged**